packages/react-core/src/components/Select/Select.tsx

```diff
@@ -70,6 +70,10 @@ export interface SelectProps extends MenuProps, OUIAProps {
   role?: string;
   /** Additional properties to pass to the popper */
   popperProps?: SelectPopperProps;
+  /** Height of the menu */
+  menuHeight?: string;
+  /** Maximum height of menu */
+  maxMenuHeight?: string;
```

Comment:
Could we also expose the `isScrollable` prop to pass to Menu, since that would have to be used when the actual menu height is greater than the menuHeight/maxMenuHeight props? It can be passed in fine, just isn't exposed in Select's prop table.

Also, I think it was mentioned maybe during the backlog meeting, but do we want to expose these props in Dropdown as well? Not a blocker here, but would be worth doing at some point

Reply:
Yup.  I can open a dropdown.  I will add the additional prop.

Reply:
@thatblindgeye technically they can spread the `isScrollable`, but i can add it so it is documented.

Reply:
Opened follow up issue for dropdown #10083